contracts/NXMDSValue.sol

```diff
@@ -18,30 +18,30 @@ import "./external/openzeppelin-solidity/math/SafeMath.sol";
 
 
 contract Aggregator {
-    function currentAnswer() public view returns (int); 
+    function latestAnswer() public view returns (int); 
 }
 
 
 contract NXMDSValue {
 
     using SafeMath for uint;
 
-    /// @dev Get ETH-USD feed from Chainlink and convert it to bytes32.
-    /// @return Returns ETH-USD rate in wei. 
+    /// @dev Gets DAI-ETH feed from Chainlink and converts it to ETH-DAI(in bytes32).
+    /// @return Returns ETH-DAI rate in wei. 
```

Comment:
// @dev Get DAI-ETH feed from Chainlink and convert it to ETH-DAI (in bytes32).
// @return Return rate in wei.